### DISCUSSION TITLE
Cant solve ModuleNotFoundError: No module named 'packaging'

### DISCUSSION THREAD

**Discussion by ghost** [TomSchimansky/CustomTkinter#2060] [General]
I have a clean installation of VS Code (1.84) on WIndows 11.
I have created a new venv using Python 3.12
I have installed packages into venv as follows:
customtkinter 5.2.1
darkdetect    0.8.0
Pillow        10.1.0
pip           23.2.1

I have activated the venv.

But still have the 'not found' error.
Could anyone who has solved this let me know how.

Thanks

**Comment by ghost** [comment]
Ok, I sorted by copying /pasting the two directories:

packaging
packaging-23.2.dist-ini

into the  .venv\Lib\site-packages directory.

I got these two directories from a Git cloned copy of the CustomTkinter repository.

**Reply by Akascape** [reply]
@Goodwin6322 install the packaging module with pip inside venv.

**Reply by ghost** [reply]
Thanks, that easier.
